cmd/lakefs/cmd/migrate.go

```diff
@@ -115,10 +113,8 @@ func DoMigration(ctx context.Context, kvStore kv.Store, cfg *config.Config, forc
 		switch {
 		case version >= kv.NextSchemaVersion || version < kv.InitialMigrateVersion:
 			return fmt.Errorf("wrong starting version %d: %w", version, kv.ErrMigrationVersion)
-		case version < kv.ACLNoReposMigrateVersion:
-			err = migrations.MigrateToACL(ctx, kvStore, cfg, logging.ContextUnavailable(), version, force)
-		case version < kv.ACLImportMigrateVersion:
-			err = migrations.MigrateImportPermissions(ctx, kvStore, cfg)
+		default: // ACL migration code is no longer needed since we removed ACLs
```

Comment:
I would be happier if we logged a message first to migrate using some earlier version, and maybe even failed.  It will only trigger for someone with a pre-ACL RBAC version, I guess - but it is still relevant.

Reply:
Thanks, I'm not sure I entirely understand the comment.
With this change, the migration process is an empty flow which just updates the KV schema version to latest until the next time we will introduce new changes to the schema.
We must maintain the same versioning conventions as to not break compatibility and we don't want to create a new version as to not require users to actively migrate.
The migration process is iterative and will run migration scenarios until reaching the latest KV schema version. In that sense there's no right version to start with (we will just run the migrations from that point) and so I don't understand where we're supposed to fail.
Same for the documentation comment - this function is supposed to replace the manual migration path, so that the user doesn't really need to know which schema version he's in and where he needs to go (as long as they use a lakeFS version with a valid KV schema version). All they need to do is run the migration and it should take care of everything for them.
I'll add more documentation to the function so it will be more clear what we are doing here